docs/contents/configuration.rst

```diff
@@ -942,6 +943,11 @@ Configuration options for UMIs:
   ``fastq_name`` with UMIs in read names or the path to a fastq file with
   UMIs for each aligned read.
 
+- ``correct_umis: [path/to/whitelist_umi.txt]``. For a restricted set of UMIs specify a
```

Comment:
Can this be expanded a bit? What is exactly a "restricted" set of UMIs? Even the Fulcrum Genomics page isn't too clear.